lesson_07/homeworks_07.py

```diff
@@ -62,10 +89,74 @@ def find_substring(str1, str2):
 print(find_substring(str1, str2)) # поверне -1
 
 # task 7
+"""
+Ігор займається фотографією. Він вирішив зібрати всі свої 232
+фотографії та вклеїти в альбом. На одній сторінці може бути
+розміщено щонайбільше 8 фото. Скільки сторінок знадобиться
+Ігорю, щоб вклеїти всі фото?
+"""
+
+def albbum_capacity_needed (page_capacity, photos_amount): 
+    """Calculate album capacity needed."""
+    pages_required = photos_amount/page_capacity
+    return pages_required
+
+print (f'Igor will need {albbum_capacity_needed(page_capacity = 8, photos_amount = 232)} pages of the album.')
```

Comment:
```suggestion
some_good_named_variable = albbum_capacity_needed(page_capacity = 8, photos_amount = 232)
print (f'Igor will need {some_good_named_variable} pages of the album.')
```
а тут навпаки, я згадував що так робити не варто і краще використовувати змінну чим "спагетті"